tests/phpunit/tests/media.php

```diff
@@ -4339,6 +4390,130 @@ public function test_wp_get_loading_optimization_attributes( $context ) {
 		}
```

Comment:
I think you can remove lines 4369-4390 from this test, since all that is already covered by the regular test above.